ibridges/session.py

```diff
@@ -239,5 +296,8 @@ def _translate_irods_error(exc) -> Exception:  # pylint: disable=too-many-return
     if isinstance(exc, CAT_INVALID_AUTHENTICATION):
         return LoginError("Cached password is wrong")
     if isinstance(exc, ValueError):
+        # PRC 2.0.0 does not make a difference between wrong password or expired pw
+        if exc.args[0] == "Authentication failed: scheme = 'pam', auth_type = None":
+            return LoginError("Cached password is expired", "Wrong password provided")
```

Comment:
I think we should just make it into one message -> "Your cached password is expired or you have provided the wrong password."

Reply:
This one is gone, solved by one generic PasswordError